sql/handler.cc

```diff
@@ -7868,6 +7868,30 @@ void handler::unlock_shared_ha_data() {
     mysql_mutex_unlock(&table_share->LOCK_ha_data);
 }
 
+// Split a string based on a delimiter.  Two delimiters in a row will not add
+// an empty string in the set.
+std::unordered_set<std::string> split_into_set(const std::string &input,
+                                               char delimiter) {
+  size_t pos;
+  size_t start = 0;
+  std::unordered_set<std::string> elems;
+
+  // Find next delimiter
+  while ((pos = input.find(delimiter, start)) != std::string::npos) {
+    // If there is any data since the last delimiter add it to the list
+    if (pos > start) elems.insert(input.substr(start, pos - start));
+
+    // Set our start position to the character after the delimiter
+    start = pos + 1;
```

Comment:
Is the case with trailing comma `delimiter` handled properly?

Reply:
yes